extensions/integration-tests/src/index.ts

```diff
@@ -19,6 +22,36 @@ const options: any = {
 
 if (suite === SuiteType.Stress) {
 	options.timeout = 7200000;	// 2 hours
+	if (process.env.StressRuntime) {
+		options.timeout = (120 + 1.2 * parseInt(process.env.StressRuntime)) * 1000; // allow sufficient timeout based on StressRuntime setting
```

Comment:
>StressRuntime [](start = 54, length = 13)

Convention is for env vars to be capitalized with underscores, STRESS_RUNTIME

And why is this different than all the rest? Seems like something like ADS_TEST_TIMEOUT would be clearer

Reply:
agree, but these variables are not being defined here but are pre-existing, we will need to cleanout and make this consistent all our test collateral which is a separate work item as it will touch more code than what is in scope for this change.